### PR TITLE
added screen api

### PR DESCRIPTION
**Pull request by Trinsdar**

### CODE REVIEW

File: felt-screen-api/src/main/java/io/github/feltmc/feltapi/mixin/screen/HandledScreenMixin.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import net.minecraft.screen.slot.Slot;
+import org.jetbrains.annotations.Nullable;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.Unique;
+
+@Mixin(HandledScreen.class)
+public class HandledScreenMixin implements HandledScreenExtension {
+
+    @Shadow @Nullable protected Slot focusedSlot;
```
Comment (unix-supremacist) [inline felt-screen-api/src/main/java/io/github/feltmc/feltapi/mixin/screen/HandledScreenMixin.java:13]: why new lines inbetween vars?

Comment (Trinsdar) [reply felt-screen-api/src/main/java/io/github/feltmc/feltapi/mixin/screen/HandledScreenMixin.java:13]: Oh oops

Comment (Trinsdar) [reply felt-screen-api/src/main/java/io/github/feltmc/feltapi/mixin/screen/HandledScreenMixin.java:13]: Oh actually blame the minecraft dev plugin

**Review by unix-supremacist — APPROVED** [review]
looks good

**Pull request merged**